src/current/v23.2/molt-fetch.md

```diff
@@ -329,7 +329,12 @@ A local file server can be used with either the [`IMPORT INTO` or `COPY FROM` mo
 
 `--direct-copy` specifies that MOLT Fetch should use `COPY FROM` to move the source data directly to CockroachDB without an intermediate store:
 
-- Because the data is held in memory, the machine must have sufficient RAM for the amount of data being moved.
+- Because the data is held in memory, the machine must have sufficient RAM for the data currently in flight: 
+
+	~~~
+	--row-batch-size * --export-concurrency * --table-concurrency
```

Comment:
Sorry, approving too fast. We lose count the average size of each row. So it should be `average size of each row * --row-batch-size * --export-concurrency * --table-concurrency`.

`--row-batch-size` is essentially the number of rows in a batch.

Reply:
Fixed, thanks!